### DISCUSSION TITLE
Updating question

### DISCUSSION THREAD

**Discussion by ShoveD01** [bangcorrupt/hacktribe#241] [Q&A] [answered]
Just to be clear, if I have an early version of hacktribe installed, how would I update to a more recent version?

**Comment by bangcorrupt** [comment, marked as answer]
Just patch the firmware and install.

[FAQ](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#how-do-i-upgrade-hacktribe)

Depending how old your current version is, the firmware update file may need to be at different paths.  If you have a `hacktribe` directory on your SD card, put the file under `hacktribe/System/`, otherwise use `electribe sampler/System/`.
